test/integration/ae_mdw_web/controllers/name_controller_test.exs

```diff
@@ -849,6 +849,27 @@ defmodule Integration.AeMdwWeb.NameControllerTest do
 
       assert Enum.sort(kbis) == kbis
     end
+
+    test "when retrieving names with tx_hash=true, it displays the extends/claims tx hashes", %{
+      conn: conn
+    } do
+      assert %{"data" => names} = conn |> get("/v2/names", tx_hash: "true") |> json_response(200)
+
+      assert Enum.all?(names, fn %{
+                                   "info" => %{
+                                     "claims" => claims,
+                                     "updates" => updates,
+                                     "transfers" => transfers
+                                   }
+                                 } ->
+               Enum.all?(claims ++ updates ++ transfers, fn tx_hash ->
+                 match?(
+                   {:ok, _encoded_tx_hash},
+                   :aeser_api_encoder.safe_decode(:tx_hash, tx_hash)
```

Comment:
It could be checked that besides present the `tx_hash` is correct, here on integration or on unit tests. 

An integration test for oracle would be missing or if the `tx_hash` becomes validated on oracle unit test it would be enough.

Reply:
Done, added oracle test which validates this now